contributing.md

```diff
@@ -1,9 +1,109 @@
 # Contributing
 
-When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other
-method with the owners of this repository before making a change.
+Heya! Welcome to Directus, and thank you for taking the time to contribute back to Open Source Software! ❤️ We want
+everybody to be able to contribute to Directus, no matter your background or expertise. In order to facilitate that,
+we've put together a couple ticks and tricks below. Our team truly appreciates every single contributor, community
+member, GitHub star, pull-request, bug report, and feature request. Keeping Directus completely free and open-source is
+our way of saying: **Thank you!**
 
-Please note we have a [code of conduct](./code_of_conduct.md), please follow it in all your interactions with the
+> We're here to help!
+>
+> If you have _any_ questions along your contributor journey, please feel free to come chat with us on
+> [our Discord server](https://directus.chat).
+
+## Code of Conduct
+
+**The Directus [Code of Conduct](https://github.com/directus/directus/blob/main/code_of_conduct.md) is one of the ways
+we put our values into practice. We expect all of our staff, contractors and contributors to know and follow this
+code.**
+
+**Our contributors and maintainers work extremely hard to build Directus as premium open-source software. Please be
+respectful of those efforts throughout our ecosystem. Trolling, harassing, insulting, or other unacceptable behavior by
+participants will not be tolerated.**
+
+In the interest of fostering an open and welcoming environment, we as pledge to make participation in our project and
```

Comment:
I believe the original excerpt is `we as contributors and maintainers pledge to making participation` as per the Code of Conduct:

https://github.com/directus/directus/blob/2adbb3bfd27639b3af4453d4ffcdd3be59736dea/code_of_conduct.md?plain=1#L5

but it might just be unintentionally left out in the copy/paste process 👍

Reply:
I copied this over from the live docs where the same typo exists apparently 

<img width="795" alt="CleanShot 2022-11-01 at 10 47 24@2x" src="https://user-images.githubusercontent.com/9141017/199261980-2911f347-f9f7-4408-88a4-5c0340afaeda.png">